client/src/App.tsx

```diff
@@ -82,22 +100,44 @@ function App() {
 	const [figures, setFigures] = useState<IFigure[]>([]);
 
 	const sounds: ISound = {
+		// Instrument
 		drum: drumBeat,
 		cymbal: cymbalHit,
 		guitar: guitarStrum,
-		meme: gotEm
+		trumpet: trumpet,
+		gong: gong,
+		harp: harp,
+		// Funny
+		meme: gotEm,
+		noice: noice,
+		stop_it: stop_it_get_some_help,
+		ahh: ahh,
+		air: air,
+		applause: applause,
+		groan: groan,
+		clang: clang,
+		horn: horn,
+		laugh: laugh,
+		// Nature
+		bee: bee,
+		dog: dog,
+		flying_fox: flying_fox,
+		lightning: lightning,
+		nature: nature,
+		sealion: sealion,
+		water: water
```

Comment:
related to comment above, this should be imported from another file. Also, we don't need to have this in the inside the component since it is static. this is being redefined on every render of the component